auxiliary/notes.tex

```diff
@@ -4,11 +4,42 @@
 This version of the book has been substantially revised
 since the first release in June 2019
 with feedback from readers and other experts.
-It now contains most of the major content
+We especially want to recognize the contributions of
+Arianna Legovini, for her leadership at DIME and unending support of our work, 
+as well as her detailed comments on this book, and 
+Florence Kondylis, for her leadership in founding and growing the DIME Analytics team
+and supporting this project from the very first.
+ 
+We want to thank the following members 
+of the DIME Analytics team for their contributions 
+to the ideas in this book and their help organizing them here: 
+Roshni Khincha, Avnish Singh, Radhika Kaul.
+Mizuhiro Suzuki, Yifan Powers, and Maria Arnal Canudo.
+
+Our graditude to the many people who read and offered feedback as the book took shape:
+%Alphabetical by last
+Stephanie Annijas,
+Maria Camila Ayala Guerrero,
+Thomas Escande,
+Aram Gassama,
+Steven Glover,
+Nausheen Khan,
+Michael Orevba,
+Caio Piza,
+Francesco Raffaelli,
+Daniel Rogger,
+Ankriti Singh,
+Ravi Somani,
+and Leonardo Viotti.
+And, although they number far too many to name individually, 
+we thank all the members of DIME and its teams across all the years
+for the innovative work they have done, the lessons learned,
+and the team spirit that makes our work so fruitful and rewarding.
+
+This edition of the book now contains most of the major content
```

Comment:
the sentence at the beginning seems to fit very nicely here, then the current sentence would revise to 'It now contains ...'